plan/views.py

```diff
@@ -1,33 +1,32 @@
-from django.db.models import Q
-
-from courses.models import Requirement
 from courses.views import CourseDetail, CourseList
 
+from .filters import bound_filter, requirement_filter
 from .search import TypedSearchBackend
 from .serializers import CourseDetailWithReviewSerializer, CourseListWithReviewSerializer
```

Comment:
No relative imports please

Reply:
is that a rule we could add to flake8?

Reply:
There doesn't seem to be an easy way to do that in either `flake8` or `isort`, but I'll keep looking